extensions/amp-a4a/0.1/test/test-amp-a4a.js

```diff
@@ -403,7 +403,9 @@ describe('amp-a4a', () => {
         const doc = fixture.doc;
         const a4aElement = doc.createElement('amp-a4a');
         const a4a = new AmpA4A(a4aElement);
-        return a4a.maybeRenderAmpAd_(null).then(rendered => {
+        a4a.adUrl_ = 'http://foo.com';
+        a4a.maybeRenderAmpAd_ = function() { return Promise.resolve(false); }
```

Comment:
Oh, ok.  I see what you're saying.  This will end up changing in the future, as we tighten up the promise chain regardless.  For the moment, though, your fix LGTM.